src/docker/workflows/cp-schema-registry.test.ts

```diff
@@ -132,9 +123,7 @@ describe("docker/workflows/cp-schema-registry.ts ConfluentPlatformSchemaRegistry
 
   it("start() should get the imageTag from workspace configuration", async () => {
     const customTag = "7.0.0";
-    getConfigurationStub.returns({
-      get: sandbox.stub().withArgs(LOCAL_KAFKA_IMAGE_TAG).returns(customTag),
-    });
```

Comment:
This should have never worked, since the local SR workflow doesn't use the Kafka image 🤔